plugins/encrypted-kubeconfig.yaml

```
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: encrypted-kubeconfig
+spec:
+  version: "v1.0.1"
+  # 'homepage' usually links to the GitHub repository of the plugin
+  homepage: 
+  shortDescription: "Authentication plugin for encrypted KUBECONFIG files."
```

is this the same meaning? if so let's use that. words like `plugin` are useless. Everything is a plugin here. :)
```suggestion
  shortDescription: "Encrypt KUBECONFIG files"
```